tests/sample_qc/test_de_novo.py

```diff
@@ -102,55 +151,142 @@ def test_calculate_de_novo_post_prob(
     @pytest.mark.parametrize(
         "locus, alleles, proband_expr, father_expr, mother_expr, is_xx_expr, freq_prior_expr, expected_exception, expected_result",
         [
-            # Case 1: Multiple fail conditions
+            # 1. Autosomal locus with HIGH confidence
             (
                 hl.locus("chr1", 10000, reference_genome="GRCh38"),
                 hl.literal(["A", "C"]),
-                hl.struct(GT=hl.call(0, 1), AD=[9, 2], DP=11, GQ=2, PL=[2, 0, 230]),
-                hl.struct(GT=hl.call(0, 0), AD=[0, 0], DP=0, GQ=0, PL=[0, 0, 0]),
-                hl.struct(GT=hl.call(0, 0), AD=[0, 0], DP=0, GQ=0, PL=[0, 0, 0]),
+                hl.struct(GT=hl.call(0, 1), AD=[5, 30], DP=35, GQ=99, PL=[99, 0, 1]),
+                hl.struct(GT=hl.call(0, 0), AD=[20, 0], DP=20, GQ=60, PL=[0, 60, 120]),
+                hl.struct(GT=hl.call(0, 0), AD=[25, 0], DP=25, GQ=80, PL=[0, 80, 150]),
                 hl.literal(True),
                 hl.literal(1e-5),
                 False,
                 hl.struct(
                     is_de_novo=True,
-                    p_de_novo=hl.missing(hl.tfloat64),
-                    confidence=hl.missing(hl.tstr),
-                    fail_reason=hl.set(
-                        {
-                            "min_de_novo_p",
-                            "min_proband_ab",
-                            "min_proband_gq",
-                            "parent_sum_ad_0",
-                        }
-                    ),
+                    p_de_novo=0.999,
+                    confidence="HIGH",
+                    fail_reason=hl.missing(hl.tset(hl.tstr)),
+                ),
+            ),
+            # 2. Autosomal locus with MEDIUM confidence
+            (
+                hl.locus("chr1", 11000, reference_genome="GRCh38"),
+                hl.literal(["CT", "C"]),
```

Comment:
this is the only test that has an indel -- could you switch at least one more test to be an insertion or deletion (just to balance the tests between SNVs and indels a bit more)